### PR TITLE
Code block: preserve indentation on paste

### PR DESCRIPTION
**Pull request by ellatrix**
## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/blocks/src/api/raw-handling/paste-handler.js [hunk 1]
```diff
@@ -56,7 +57,12 @@ function filterInlineHTML( HTML ) {
 	HTML = removeInvalidHTML( HTML, getPhrasingContentSchema( 'paste' ), {
 		inline: true,
 	} );
-	HTML = deepFilterHTML( HTML, [ htmlFormattingRemover, brRemover ] );
+
+	console.log( preserveWhiteSpace );
```
Comment (getdave) [inline packages/blocks/src/api/raw-handling/paste-handler.js:61]: Assume you want to ditch this?

File: packages/blocks/src/api/raw-handling/paste-handler.js [hunk 2]
```diff
@@ -140,7 +147,9 @@ export function pasteHandler( {
 	plainText = '',
 	mode = 'AUTO',
 	tagName,
+	preserveWhiteSpace,
 } ) {
+	console.log( { preserveWhiteSpace } );
```
Comment (getdave) [inline packages/blocks/src/api/raw-handling/paste-handler.js:152]: We'll want to remove this.

**Review by getdave — COMMENTED** [review]
Thanks for fixing this. I haven't had a chance to manually test just yet, but noticed some stray `console.log`s...just in case.

File: packages/blocks/src/api/raw-handling/paste-handler.js (hunk 1 above)
Comment (ellatrix) [reply packages/blocks/src/api/raw-handling/paste-handler.js:61]: Yeah, sorry, had to push this quick

Comment (ellatrix) [reply packages/blocks/src/api/raw-handling/paste-handler.js:61]: Strange the linter didn't catch it

**Review by ntsekouras — APPROVED** [review]
Tested and looks good, thanks!

**Review by aristath — APPROVED** [review]

**Pull request merged**